scripts/deployments/deploy.sh

```diff
@@ -292,26 +292,40 @@ if [ "$start_simcore" -eq 0 ]; then
     fi
 fi
 # shellcheck disable=2235
-if [ "$start_opsstack" -eq 0 ] && ([ "$stack_target" = "dalco" ] || [ "$stack_target" = "master" ] || [ "$stack_target" = "public" ]); then
+
+# We pause 2 mns to be sure the deployment agent is launched
+sleep 120
```

Comment:
Could we live without this? A 2 min sleep without any log-message telling the operator that the script will now sleep will really look like there is some bug. I prefer not to have any sleeps outside retrying-functionality (the while loops you made below are retrying, so there it is fine)

Reply:
Done